test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -4576,6 +4576,61 @@ BOOST_AUTO_TEST_CASE(constructing_enums_from_ints)
 	ABI_CHECK(callContractFunction("test()"), encodeArgs(1));
 }
 
+BOOST_AUTO_TEST_CASE(struct_referencing)
+{
+	static char const* sourceCode = R"(
+		pragma experimental ABIEncoderV2;
+		interface I {
+			struct S { int a; }
+		}
+		library L {
+			struct S { int a; }
```

Comment:
Please change this to `uint b; uint a;` so that the test has a chance of catching the difference.